include/libdnf/transaction/transaction_item_action.hpp

```diff
@@ -31,13 +31,13 @@ namespace libdnf::transaction {
 // TODO(jmracek) Copy enum to SWDB including their string definitions and already removed values to keep compatibility
 // Then transform old database data to transform stored reverse action to REPLACED action and remove all unused action from code
 enum class TransactionItemAction : int {
-    INSTALL = 1,         // a new package that was installed on the system
-    DOWNGRADE = 2,       // an older package version that replaced previously installed version
-    UPGRADE = 6,         //
-    REMOVE = 8,          // a package that was removed from the system
-    REINSTALL = 9,       // a package that was reinstalled with the identical version
-    REASON_CHANGE = 11,  // a package was kept on the system but it's reason has changed
-    REPLACED = 12        // Replace actions UPGRADED, DOWNGRADED, REINSTALLED, OBSOLETED
+    INSTALL = 1,
+    UPGRADE = 2,
+    DOWNGRADE = 3,
```

Comment:
Why did you remove comments?

Reply:
I felt they are redundant in that they just re-iterated what was already clear from the enum value names. I kept comments for values where it wasn't entirely obvious.

Reply:
For example install is also used for install only packages even when a package is installed. More comments is a better than less.

Reply:
> For example install is also used for install only packages even when a package is installed.

Well, this was not mentioned in the comment I have removed, in fact from that standpoint the original comment could be somewhat misleading, but I'm not sure this is the place for explaining the various nuances and special cases.

> More comments is a better than less.

Depending on how you mean it. Comments that don't bring any new information and only reiterate what is obvious from the code are not better, IMO they are worse, as they add clutter and don't really help in any way.

In this case, anyone having a broad understanding of libdnf code will know this enum is for and having a comment like:
```
    INSTALL,  // a new package is being installed
```
Will not bring any new information to them. I can put the comments back if you want, it doesn't matter much. But I think understanding the above is important, I often see trivial comments that are not useful and then when there's a complex non-obvious interaction, it has no comment explaining it.